### ISSUE TITLE
[Request] gtk#

### ISSUE THREAD

**Issue by rihaq** [ScoopInstaller/Main#3791]
### Criteria

For a package to be acceptable in the Main bucket, it should be:

- [ ] a non-GUI tool
- [ ] a reasonably well-known and widely used developer tool (e.g. if it's a GitHub project, it should have at least 500 stars and 150 forks)
- [ ] the latest stable version of the program
- [ ] the full version i.e. not a trial version
- [ ] a fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Information

**Name:** [name of new package]
Gtk#
**Description:** [clear and concise details of what it is]
Gtk# is a Graphical User Interface Toolkit for mono and .Net
**Homepage:** [a URL/link]
https://www.mono-project.com/gtksharp
**Download link(s):** [URL(s)/link(s)]
https://xamarin.azureedge.net/GTKforWindows/Windows/gtk-sharp-2.12.45.msi
**Some indication of popularity/repute:** [GitHub stars/software reviews etc.]

406

**Comment by rashil2000** [comment]
This looks like a GUI application? It should go into Extras bucket.

**Comment by tech189** [comment]
Yes I think so. And the correct homepage url is https://www.mono-project.com/docs/gui/gtksharp/
